searx/infopage/en/privacy-policy.md

```diff
@@ -0,0 +1,217 @@
+
+  - Your query;
+  - The language you selected on the SearXNG instance;
+  - The technical parameters are needed to form the same query.
+
+2. **No metadata is transmitted**.
+
+**In the end, only**:
+
+- your query,
+- the language you selected on the SearXNG instance and
+- the technical parameters needed to form the same query
+
+are transmitted to search engines.
+
+## Can the search engines see your data or install cookies or other stuff on your browser?
+
+**The answer is: No, they cannot.**
+
+Indeed, the search engines (for example, DDG or qwant) that receive your query
+through the SearXNG instance:
+
+1. Do not know from where (IP or location) your query comes and see your IP
+   since they are talking only with the SearXNG instance;
+
+2. Acquire only the IP of the SearXNG instance you used for your query.
+
+3. Cannot send your browser cookies or other data to track you since the search
+   engines that receive your query are only connected to the SearXNG instance
+   and not your web browser.
+
+## What data is collected
+
+This SearXNG instance receives only your IP but does not collect it.
+
+**Each user is responsible for the content they intend to submit as a query**.
+
+### Who can access the data, and for what activities?
+
+None neither the server administrator (instance) can access Personally
+Identifiable Information (PII) nor data of the queries but only - for technical
+needs - system logs without the possibility of retrieving any personal data
+anyway.
+
+## The purposes of the processing
+
+When data subjects or SearXNG providers are in the EEA, the GDPR applies.
+Still, the purpose is to provide all access to the SearXNG instance by allowing
+users to submit queries and read and consult the search results.
+
+Furthermore, the purposes are also related to server maintenance and system and
+application upgrades.
+
+The optional, explicit, and voluntary sending of electronic mail to the
+addresses indicated on the footer of this site involves the acquisition of the
+sender's address necessary for the replies and any other personal data contained
+in the message.  These data are processed to respond to messages sent and handle
+related requests. Failure to provide personal data for communications with us or
+send requests will prevent evading them. We store data for the time strictly
+necessary for the purposes related to data processing.
+
+## Legal basis for the processing
+
+When data subjects or SearXNG providers are in the EEA, the GDPR applies.
+Still, the processing of personal data is based on consent - according to
+Article 6, par. 1, letter a) of EU Regulation 2016/679 - expressed by the user
+by browsing this website, choosing the preferences, and submitting queries, thus
+accepting this information.
+
+Consent is optional, and the user can withdraw at any time by request sent by
+email to `{maintainer's email}`¡, specifying that, in this case, whether the
```

Comment:
Same here, see above https://github.com/searxng/searxng/pull/2195/files#r1108777544

Can we reword the two sentences so that they are more general and only refer to a contact opportunity?

What do we do if the maintainer has not configured a contact address (optional)?

Reply:
Please take a look at my comment above.

Every maintainer has the responsibility to inform the data subjects.